tldr.py

```diff
@@ -468,7 +468,7 @@ def main():
             if not result:
                 sys.exit((
                     "`{cmd}` documentation is not available. "
-                    "Consider contributing Pull Request to "
+                    "If you want to contribute it, feel free to send a pull request to:"
```

Comment:
I might need to add a new line like the rest

Reply:
Yeah, add a newline after just the first line.

Reply:
Will do!

Reply:
https://github.com/tldr-pages/tldr-python-client/pull/163/commits/654e7600d4110705c9c2ab43ce1a80caeaf698c1